pkgs/development/tools/misc/itmdump/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, rustPlatform, pkg-config }:
+
+rustPlatform.buildRustPackage rec {
+  version = "0.3.1";
+  pname = "itm";
+
+  src = fetchFromGitHub {
+    owner = "rust-embedded";
+    repo = "itm";
```

Comment:
It seems that `itmdump` was removed from that repository and has moved to:

https://github.com/japaric/itm-tools

So this derivation should probably use that instead (there are no releases yet, so the version could be date-based). The attribute name and derivation directory could then be renamed to `itm-tools`.